Makefile

```diff
@@ -264,6 +264,15 @@ clean-release-git: ## Restores the git files usually modified during a release
 clean-release: clean-release-git
 	rm -rf $(RELEASE_DIR)
 
+.PHONY: clean-child-clusters
+clean-child-clusters: $(KUBECTL)
+	@for cluster in $$($(KUBECTL) get clusters -o name); do \
```

Comment:
Should be able to do this I think?
`$(KUBECTL) delete clusters -A --all`
Same below for VPCs

Reply:
Thanks, updated.